xml_analyzer/data_container.py

```diff
@@ -1,38 +1,72 @@
-import collections
+from enum import Enum
+from .definitions import MappingElement
+
+
+class DataContainerFields(Enum):
+    variables = 'variables'
+    autocalc = 'autocalc'
 
 
 class DataContainer:
```

Comment:
could be a [dataclasss](https://docs.python.org/3/library/dataclasses.html)

Reply:
Interesting. Changed to data class, but what is the advandage over enum class in this case?

Reply:
Changed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/a01925bac39c4c81ffc60b598221bd35835a36d1

Reply:
I was talking about DataContainer class

Reply:
Fixed with